### DISCUSSION TITLE
Interlacing monitors

### DISCUSSION THREAD

**Discussion by rohanrichards** [debauchee/barrier#1601] [Q&A]
I've been through issues and I'm fairly certain this is a proposed change that hasn't come to light yet, but I just want to confirm explicitly.

Is this:
![image](https://user-images.githubusercontent.com/16222002/157132795-639d7bb7-716b-4915-8131-5802d5591c4c.png)
Possible with Barrier?

My setup is 
[win10-1920x1080] [win10-5140x1440] [win10-1920x1080]

My goal is to use PbP on the middle monitor, and run a laptop in one of the splits, so it would work something like this:
[win10-1920x1080] [linux-2570x1440] [win10-2570x1440] [win10-1920x1080]

Secondly, if this isn't possible with Barrier (I've confirmed it's not possible on Synergy), can anyone suggest a solution?

**Comment by Fr4cktail** [comment]
did you find a solution?

**Reply by rohanrichards** [reply]
Not an ideal solution, I am still just using Synergy. I have disabled edge switching and just use a keyboard shortcut to switch between devices.

**Reply by xantrk** [reply]
@rohanrichards I am on a very similar boat, using a combination of autohotkey and ClickMonitor (to command output switch to my monitor) but it is quite buggy as barrier doesn't play nice with multi key combos, sometimes they recognized as stuck. How are you doing this exactly?
